packages/material-ui/src/ButtonGroup/ButtonGroup.test.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { assert } from 'chai';
+import {
+  createMount,
+  createShallow,
+  createRender,
+  // describeConformance,
+  getClasses,
+} from '@material-ui/core/test-utils';
+import Button from '../Button/Button';
+import ButtonGroup from './ButtonGroup';
+
+describe('<ButtonGroup />', () => {
+  let mount;
+  let shallow;
+  let render;
+  let classes;
+
+  before(() => {
+    mount = createMount({ strict: true });
+    shallow = createShallow({ dive: true });
+    render = createRender();
+    classes = getClasses(
+      <ButtonGroup>
+        <Button>Hello World</Button>
+      </ButtonGroup>,
+    );
+  });
+
+  after(() => {
+    mount.cleanUp();
+  });
+
+  // describeConformance(<ButtonGroup><Button>Conformance?</Button></ButtonGroup>, () => ({
+  //   classes,
+  //   mount,
+  //   refInstanceof: window.HTMLDivElement,
+  //   skip: ['componentProp'],
+  // }));
+
+  it('should render with the root class but no others', () => {
+    const wrapper = shallow(
+      <ButtonGroup>
+        <Button>Hello World</Button>
+      </ButtonGroup>,
+    );
+    assert.strictEqual(wrapper.hasClass(classes.root), true);
+    assert.strictEqual(wrapper.hasClass(classes.contained), false);
+    assert.strictEqual(wrapper.hasClass(classes.fullWidth), false);
+    assert.strictEqual(wrapper.childAt(0).hasClass(classes.grouped), true);
```

Comment:
We should avoid these tests for new components. Instead try finding DOM element s by their respective aa11y selector i.e. `wrapper.find('[role="group"]').hasClass(classes.contained)` or `wrapper.find('button').hasClass(classes.group)`.